astroid/tree/base.py

```diff
@@ -107,6 +127,10 @@ def __str__(self):
             lines = pprint.pformat(value, indent=2,
                                    width=width).splitlines(True)
 
+            # Some fields, notably source_code for Module nodes, are
+            # too long to display comfortably, so this limits them.
+            if len(lines[0]) > 160:
```

Comment:
Let's hardcode this value somewhere as a global variable, rather than directly in this function.